lib/Echidna/Transaction.hs

```diff
@@ -119,25 +117,23 @@ mutateTx t@(Tx (SolCall c) _ _ _ _ _ _) = do f <- oftenUsually skip mutate
                                                  skip    _ = pure t
 mutateTx t                              = pure t
 
--- | Lift an action in the context of a component of some 'MonadState' to an action in the
--- 'MonadState' itself.
-liftSH :: (MonadState a m, Has b a) => State b x -> m x
-liftSH = stateST . runState . zoom hasLens
-  -- This is the default state function written in terms of get and set:
-  where stateST f = do
-          s <- get
-          let ~(a, s') = f s
-          put s'
-          return a
+
+-- | Transform an EVM action from HEVM to our MonadState VM
+fromEVM :: MonadState VM m => EVM a -> m a
```

Comment:
Should we move this to a different file?

Reply:
what about moving it to `Echidna.Types`?

Reply:
Sounds good.